plugin/src/main/java/org/wildfly/plugin/provision/AbstractProvisionServerMojo.java

```diff
@@ -195,6 +202,9 @@ public void execute() throws MojoExecutionException, MojoFailureException {
             getLog().debug(String.format("Skipping " + getGoal() + " of %s:%s", project.getGroupId(), project.getArtifactId()));
             return;
         }
+        if (dryRun) {
+            getLog().info("Dry run execution, no server provisioned.");
```

Comment:
Minor, but I think we should say something like "Dry run execution, no server will be provisioned"

Reply:
Ok, thanks.